OpenGraphMeta.class.php

```diff
@@ -61,21 +63,21 @@ public static function setMainTitlePF( Parser $parser, $mainTitle ) {
 	/**
 	 * @param OutputPage &$out
 	 * @param ParserOutput $parserOutput
+	 * @throws ConfigException
+	 * @throws FatalError
+	 * @throws MWException
 	 */
 	public static function onOutputPageParserOutput( OutputPage &$out, ParserOutput $parserOutput ) {
-		global $wgLogo, $wgSitename, $wgXhtmlNamespaces, $egFacebookAppId, $egFacebookAdmins;
-
+		global $wgLogo, $wgSitename, $wgXhtmlNamespaces;
+		$egFacebookAppId = MediaWikiServices::getInstance()->getMainConfig()->get( 'egFacebookAppId' ); //FandomChange
+		$egFacebookAdmins = MediaWikiServices::getInstance()->getMainConfig()->get( 'egFacebookAdmins' ); //FandomChange
 		$setMainImage = $parserOutput->getExtensionData( 'setmainimage' );
 		$setMainTitle = $parserOutput->getExtensionData( 'setmaintitle' );
 
 		if ( $setMainImage !== null ) {
-			if ( method_exists( MediaWikiServices::class, 'getRepoGroup' ) ) {
```

Comment:
why do we change that?

Reply:
that one I can revert, I copied this from previous PR and I see that this was not our change but they changed it in beetween